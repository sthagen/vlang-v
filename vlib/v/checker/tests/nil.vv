fn main() {
    unsafe {
        value := nil
        println(value)
    }
}
