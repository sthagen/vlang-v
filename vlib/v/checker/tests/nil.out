vlib/v/checker/tests/nil.vv:3:18: error: use of untyped nil in assignment
    1 | fn main() {
    2 |     unsafe {
    3 |         value := nil
      |                  ~~~
    4 |         println(value)
    5 |     }
